app/javascript/components/upload/UploadWizard.js

```diff
@@ -22,10 +23,11 @@ import CoordinateLabelStep from './CoordinateLabelStep'
 import RawCountsStep from './RawCountsStep'
 import ProcessedExpressionStep from './ProcessedExpressionStep'
 
+const CHUNK_SIZE = 10000000
 const STEPS = [RawCountsStep, ProcessedExpressionStep, ClusteringStep, CoordinateLabelStep, ImageStep]
 
 /** shows the upload wizard */
-export default function UploadWizard({ accession, name }) {
+export default function UploadWizard({ studyAccession, name }) {
```

Comment:
FWIW, `accession` strikes me as snappier than `studyAccession`, and I somewhat wish we'd use the former at the start of our big JS refactor over a year ago.